testing.go

```diff
@@ -1,13 +1,22 @@
 package amqprpc
 
 import (
+	"encoding/json"
+	"fmt"
 	"net"
+	"net/http"
+	url2 "net/url"
```

Comment:
Haha, instead of changing the variable name? Or is this IntelliJ trying to be smart? :)